### PR TITLE
Ported Prometheus/Grafana SLO Demo

### PR DESCRIPTION
**Pull request by rewsmith**
Ported across from [demo-slo-prometheus-grafana](https://github.com/TykTechnologies/demo-slo-prometheus-grafana).  Changes made to avoid port collisions and also modified the directory structure to fall inline with Tyk Demo standards

### CODE REVIEW

**Comment by sedkis** [comment]
@carlostyk  - mind having a play with this PR please?  I believe you have some experience playing with this stack

**Comment by carlostyk** [comment]
@sedkis @rewsmith  - always ready to help!
I am missing something in the bootstrap logs - I would also put the user and password of the Grafana user.  To be consistent with the other part of the deployment. What do you think?

File: deployments/slo-prometheus-grafana/bootstrap.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+fi
+log_ok
+bootstrap_progress
+
+log_end_deployment
+
+echo -e "\033[2K
+▼ SLO Prometheus Grafana
+  ▽ Grafana
+                    URL : http://localhost:3020/
```
[the remark this answers is not in the document]
Comment (rewsmith) [reply deployments/slo-prometheus-grafana/bootstrap.sh:45]: Tis done

File: deployments/slo-prometheus-grafana/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+```
+
+You will see K6 output in your terminal:
+
+<img width="963" alt="K6" src="https://user-images.githubusercontent.com/17831497/187454878-c1182107-7de2-4cd0-b572-9bcc80dde6c3.png">
+
+
+3. Check out the dashboard in Grafana
+
+Go to [Grafana](http://localhost:3000/) in your browser (initial user/pwd: admin/admin) and open the dashboard called [*SLOs for APIs managed by Tyk*](./volumes/grafana/dashboards/SLOs-for-APIs-managed-by-Tyk.json).
```
[the remark this answers is not in the document]
Comment (rewsmith) [reply deployments/slo-prometheus-grafana/README.md]: Tis done

**Comment by rewsmith** [comment]
@sedkis @carlostyk @davegarvey Comments addressed and ready for review again

**Review by carlostyk — APPROVED** [review]
Thank you @rewsmith for the integration!!!

**Comment by sedkis** [comment]
Tested on m1, works perfect, thx

**Comment by davegarvey** [comment]
Thanks for this contribution Andy 👍

**Pull request merged**